R/ui.R

```diff
@@ -7,106 +7,111 @@ source(paste0(UICompDirectory, "policies.R"), local = TRUE)
 source(paste0(UICompDirectory, "dashboard.R"), local = TRUE)
 
 shinyUI(
-    dashboardPage(
-        dashboardHeader(title = "Gradebook"),
-        dashboardSidebar(
-            sidebarMenu(
-                tags$head(
-                    tags$style(HTML('
-                    .fixed-width-icon {
-                      width: 25px !important;
-                      text-align: left;
-                    }
-                '))
-                ),
-                menuItem("Dashboard", tabName = "dashboard", icon = icon("chart-line", class = "fixed-width-icon")),
-                #  menuItem("Files", tabName = "files", icon = icon("folder", class = "fixed-width-icon")),
-                menuItem("Policies", tabName = "policies", icon = icon("file-pen", class = "fixed-width-icon")),
-                hr(),
-                div(class = "sidebar-text",
-                    icon("upload", style = "margin-left: 20px;"),
-                    "Upload Student Data"
-                ),
-                fileInput("upload_gs", label = NULL, accept = c(".csv")),
-                hr(),
-                div(class = "sidebar-text",
-                    icon('upload', style = "margin-left: 20px;"),
-                    "Upload Policy File"
-                ),
-                fileInput("upload_policy", label = NULL, accept = c(".yml")),
-                hr(),
-                div(style = "display: flex; align-items: center; margin-left: 20px;",
-                    h5("Download Files")
-                ),
-                div(style = "display: flex; align-items: center; margin-left: 15px;",
-                    div(style = "flex-grow: 1; margin-right: 10px;", 
-                        downloadButton("download_grades", "Grades", style = "width: 100%;")),
-                    div(style = "flex-grow: 1; margin-right: 10px;", 
-                        downloadButton("download_policy_file", "Policy File", style = "width: 100%;"))
-                ),
-                div(style = "display: flex; align-items: center; margin-left: 15px; color: #007bff;",
-                    
-                )
-            )
+  dashboardPage(
```

Comment:
<img width="519" alt="image" src="https://github.com/user-attachments/assets/3d210f94-8b9c-48af-8b44-f1b486abbe9b">

You need to add it here.